frontend/apps/reader/modules/readerview.lua

```diff
@@ -1005,7 +1092,10 @@ function ReaderView:shouldInvertBiDiLayoutMirroring()
     return self.inverse_reading_order and G_reader_settings:isTrue("invert_ui_layout_mirroring")
 end
 
+-- if dual page is enabled for cbz, then readerpagging will set the correct base page.
```

Comment:
There is a case where readerpagging is not the one sending the event, e.g. Directly going to a page.
So we might actually need to calculate what the base is here, and not assume it came from a relative movement.

E.g. go to page 3 should result in base 2. ATM this is not the case.